scripts/99-thrift.yaml

```diff
@@ -0,0 +1,20 @@
+manifest:
```

Comment:
Can this be made identical to 
https://github.com/zephyrproject-rtos/gsoc-2022-thrift/pull/141/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22 ?